pkg/nfd-master/nfd-api-controller.go

```diff
@@ -54,6 +54,8 @@ func newNfdController(config *restclient.Config, nfdApiControllerOptions nfdApiC
 	}
 
 	nfdClient := nfdclientset.NewForConfigOrDie(config)
+	klog.Infof("NFD API controller options: %v", nfdApiControllerOptions)
```

Comment:
Let's put this behind a V
```suggestion
	klog.V(2).Infof("NFD API controller options: %v", nfdApiControllerOptions)
```
having it as regular log would cause too much noice, but is good for debbuging situations

Reply:
This is only logged ad startup so V(0) is not so bad but maybe V(1) or V(2) would be better. I also sugges to use `utils.KlogDump` to get well-formatted output